### PR TITLE
SREP-2178 - Add perms for dedicated-admins to manage strimzi cluster operator

### PR DESCRIPTION
**Pull request by rogbas**
- Add permissions for dedicated-admins to deploy strimzi operator into the openshift-strimzi namespace
- Add permissions for dedicated-admins to manage the operator from that namespace

SREP-2178

### CODE REVIEW

File: deploy/osd-strimzi-operator/05-role.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  - pods/log
+  - events
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups:
+  - logging.openshift.io
+  resources:
+  - clusterloggings
```
Comment (jewzaam) [inline deploy/osd-strimzi-operator/05-role.yaml]: `ClusterLogging` CR should not be allowed in this namespace.

File: deploy/osd-strimzi-operator/05-role.yaml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  resources:
+  - "*"
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```
Comment (jewzaam) [inline deploy/osd-strimzi-operator/05-role.yaml]: Why not use "*" for the kafka verbs?

File: deploy/osd-strimzi-operator/05-role.yaml (hunk 1 above)
Comment (rogbas) [reply deploy/osd-strimzi-operator/05-role.yaml]: We grant that on codeready workspaces NS, I understood it was there to allow customers to manage logging of those components: https://github.com/openshift/managed-cluster-config/blob/master/deploy/osd-codeready-workspaces/05-role.yaml

File: deploy/osd-strimzi-operator/05-role.yaml (hunk 2 above)
Comment (rogbas) [reply deploy/osd-strimzi-operator/05-role.yaml]: Could be, it's the same thing.

Comment (rogbas) [reply deploy/osd-strimzi-operator/05-role.yaml]: Just updated verbs to "*"

**Comment by rogbas** [comment]
rebased with latest changes in the repo

File: deploy/osd-strimzi-operator/05-role.yaml (hunk 1 above)
Comment (jewzaam) [reply deploy/osd-strimzi-operator/05-role.yaml]: If we give it to crw we should remove it.  This CR will stand up a whole new logging stack.  Probably came with copying perms for the `openshift-logging` namespace.

**Comment by rogbas** [comment]
@jewzaam I cleaned up logging related permissions from other folders as well

**Pull request merged**